lua/nvim-treesitter/health.lua

```diff
@@ -142,7 +142,25 @@ function M.check()
     health_start "The following errors have been detected:"
     for _, p in ipairs(error_collection) do
       local lang, type, err = unpack(p)
-      health_error(lang .. "(" .. type .. "): " .. err)
+      local lines = {}
+      table.insert(lines, lang .. "(" .. type .. "): " .. err)
+      local files = vim.treesitter.query.get_query_files(lang, type)
+      if #files > 1 then
```

Comment:
maybe this is also informative in case of a single file? To show the file path?

Reply:
I think it makes sense.